config/istio/control-plane-network-policy.yaml

```diff
@@ -1,4 +1,16 @@
-#@ load("@ytt:data", "data")
+#! Documentation for Istio ports https://istio.io/docs/ops/deployment/requirements/#ports-used-by-istio
+#@ control_plane_monitoring = 15014 #! used by Citadel, Citadel agent, Galley, Mixer, Pilot, Sidecar Injector
```

Comment:
I moved these into `ytt` variables instead of a separate values file since it was confusing `cf-for-k8s` and because I don't think we actually want to expose this as changeable config at this point. My understanding was that we were using the values to give meaning to these random ports and to DRY it up a bit.

If we want to keep the values files, we'd have to separate `istio` into it's own "ytt library" here:
https://github.com/cloudfoundry/cf-for-k8s/blob/6482fa75155f3ab8f373da1754b244b6cf439e3d/config/networking.yml#L23